README.md

```diff
@@ -11,6 +11,8 @@ A vagrant provisioned development environment using the [development-environment
 * [Virtualbox 5.1.22](https://www.virtualbox.org/wiki/Download_Old_Builds_5_1) (newer versions do work, but we guarantee the version listed works)
 * [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
 
+Windows users will need to add the installation directory of Virtualbox to their `Path` environment variable. An initial check of installed applications needs this.
```

Comment:
An initial check of installed applications needs this.

What do you mean by this?